tests/load/metrics.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package load
+
+import (
+	"errors"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type Metrics struct {
+	txsIssuedCounter    prometheus.Counter
+	txsConfirmedCounter prometheus.Counter
+	txsFailedCounter    prometheus.Counter
+	txLatency           prometheus.Histogram
+}
+
+func NewMetrics(registry *prometheus.Registry) (*Metrics, error) {
+	m := &Metrics{
+		txsIssuedCounter: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "txs_issued",
+			Help:      "Number of transactions issued",
+		}),
+		txsConfirmedCounter: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "txs_confirmed",
+			Help:      "Number of transactions confirmed",
+		}),
+		txsFailedCounter: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "txs_failed",
+			Help:      "Number of transactions failed",
+		}),
+		txLatency: prometheus.NewHistogram(prometheus.HistogramOpts{
+			Namespace: namespace,
+			Name:      "tx_latency",
+			Help:      "Latency of transactions",
+		}),
+	}
+
+	if err := errors.Join(
+		registry.Register(m.txsIssuedCounter),
+		registry.Register(m.txsConfirmedCounter),
+		registry.Register(m.txsFailedCounter),
+		registry.Register(m.txLatency),
+	); err != nil {
+		return nil, err
+	}
+
+	return m, nil
+}
+
+func (m *Metrics) IncIssuedTx() {
```

Comment:
Should we match the tracker method names? For example `Issue`, `ObserveConfirmed`, etc.? Or we should change the tracker to match this, but it sounds strange to have different naming for effectively the same event.